webapp/sources/rudder/rudder-web/src/main/elm/sources/Editor/ViewTechniqueTabs.elm

```diff
@@ -141,46 +155,63 @@ techniqueParameter model technique param =
             , attribute "data-html" "true"
             ] [ i [ class "text-info fa fa-question-circle" ] []]
           ]
+          , invalidEnumTypeElem
```

Comment:
I think this should be moved upwards by 1 container : in the current "input-group" div, this looks like : 

![image](https://github.com/Normation/rudder/assets/65616064/cfbeaca3-755e-4bd0-99e1-e8ee897e7d2e)

Reply:
In fact, we should move it downwards :D
![image](https://github.com/Normation/rudder/assets/23410978/91b5f893-37bb-4b6d-81f3-ae9691bd60ec)